packages/salesforcedx-vscode-apex-debugger/package.nls.json

```diff
@@ -1,9 +1,15 @@
 {
-  "user_id_filter_text": "A comma-separated list of user IDs with requests whitelisted for debugging",
-  "request_type_filter_text": "A comma-separated list of ApexRequesType values whitelisted for debugging",
-  "entry_point_filter_text": "A regular expression matching entry points whitelisted for debugging",
+  "user_id_filter_text":
+    "A comma-separated list of user IDs with requests whitelisted for debugging",
+  "request_type_filter_text":
+    "A comma-separated list of ApexRequesType values whitelisted for debugging",
+  "entry_point_filter_text":
+    "A regular expression matching entry points whitelisted for debugging",
   "sfdx_project_text": "Location of SFDX project",
   "launch_snippet_label_text": "Apex Debugger: Launch",
-  "launch_snippet_description_text": "A new configuration for launching Apex Debugger",
-  "launch_snippet_name": "Launch Apex Debugger"
+  "launch_snippet_description_text":
+    "A new configuration for launching Apex Debugger",
+  "launch_snippet_name": "Launch Apex Debugger",
+  "trace_text":
+    "Produce diagnostic output. Instead of setting this to true you can list one or more selectors separated with commas. The 'all' selector enables very detailed output. Other selectors are 'protocol', 'variables', 'launch'"
```

Comment:
For the ones that are unused, do you plan on adding them in this PR or in another work item?

Reply:
They are all in use right now. New ones should be added as we go.